src/message/__tests__/fetchActions-test.js

```diff
@@ -116,13 +116,19 @@ describe('fetchActions', () => {
       jest.runAllTimers();
     });
 
-    test('retries a call, if there is an exception', async () => {
+    test('retries a call if there is a non-client error', async () => {
```

Comment:
Could you add something like the commit message as a comment here? I think that's more useful.